docs/reference/project_configuration.md

```diff
@@ -159,6 +159,32 @@ If `conda-forge` is not present in `conda-pypi-map` `pixi` will use `prefix.dev`
 conda-pypi-map = { "conda-forge" = "https://example.com/mapping", "https://repo.prefix.dev/robostack" = "local/robostack_mapping.json"}
 ```
 
+### `channel-priority` (optional)
+
+This is the setting for the priority of the channels in the solver step.
+
+Options:
+
+- `strict`: **Default**, The channels are used in the order they are defined in the `channels` list.
+    Only packages from the first channel that has the package are used.
+- `disabled`: There is no priority, all package variants from all channels will be set per package name and solved as one.
```

Comment:
```suggestion
- `disabled`: There is no priority, all package variants from all channels will be set per package name and solved as one.
  Care should be taken when using this option. 
  Since package variants can come from _any_ channel when you use this mode, packages might not be compatible. 
  This can cause hard to debug ABI incompatibilities.
  
  We strongly discourage using this option.
```